packages/pipeline-editor/src/CustomFormControls/validators/index.ts

```diff
@@ -26,6 +28,21 @@ export function getErrorMessages<T>(value: T, validators: Validator<T>[]) {
     .map((v) => v.message);
 }
 
+export const ErrorMessage = styled.div`
+  position: absolute;
+  left: 0;
+  right: 0;
+  padding: 5px;
+  box-sizing: border-box;
+  margin-top: -1px;
+  z-index: 1;
+  border-style: solid;
+  border-width: 1px;
+  border-color: ${({ theme }) => theme.palette.errorMessage.errorBorder};
+  background-color: ${({ theme }) => theme.palette.errorMessage.main};
+  color: ${({ theme }) => theme.palette.errorMessage.contrastText};
+`;
+
```

Comment:
Can we but this in a `ErrorMessage.tsx` file that get exported from here instread? otherwise lgtm